src/controllers/request.js

```diff
@@ -9,10 +9,56 @@ import { echoNotification } from '../helpers/notificationSender';
 const {
   Op, where, cast, col
 } = Sequelize;
+import lodash from 'lodash'
 
 environment.config();
-
 export default class usersController {
+  
+  static async createOneWayTrip(req, res) {
+    try {
+      const { id } = req.user;
+      const {
+        origin, destination, departureDate, reason, accommodation
+      } = req.body;
+
+      const managerId = await models.User.findOne({
```

Comment:
You should also add this in helpers for avoiding code redundancy and for the future use.